src/main/webapp/css/main.css

```diff
@@ -168,3 +168,69 @@ button.remove-user-button {
   color: white;
   font-weight: lighter;
 }
+
+.jasmineBio {
+    text-align: center;
```

Comment:
I think `text-align: center` is the only thing you need here. Maybe `line-height` too if you don't want them to be very spaced out. I prefer text a little smaller, but that's just my opinion.